### ISSUE TITLE
Crash on Exploratory Factor Analysis

### ISSUE THREAD

**Issue by guidocor** [jasp-stats/jasp-desktop#1743]
## Environment (for bugs)

* JASP version: 0.8.1.1
* OS name and version: Ubuntu 16.04
* Analysis: Exploratory Factor Analysis
## Environment (for enhancement requests)
 
Trying to perform an Exploratory Factor Analyisis it gives me an error: 
"This analysis terminated unexpectedly.
Error in loadNamespace(name): there is no package called ‘psych’
Stack trace
analysis(dataset = NULL, options = options, perform = perform, callback = the.callback)
psych::fa.parallel
getExportedValue(pkg, name)
asNamespace(ns)
getNamespace(ns)
tryCatch(loadNamespace(name), error = function(e) stop(e))
tryCatchList(expr, classes, parentenv, handlers)
tryCatchOne(expr, names, parentenv, handlers[[1]])
value[[3]](cond)
To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports"

Happens also with Principal Component Analysis. Tryied at my 3 computers (same configuration all of them) and always the same error. psych package is installed correctlly  (.libPaths(); "/usr/local/lib/R/site-library" "/usr/lib/R/site-library"  "/usr/lib/R/library")           
 and I can do the analysis in the R console.

**Comment by FransMeerhoff** [comment]
Hello @guidocor 

On Ubuntu we recommend for running JASP to execute the commands below.
(See https://jasp-stats.org/installation-guide/ Ubuntu part)
It seems that the last line of these commands is missing, because installing graph will also provide psych. Hope that installing the packages on the last line will solve your problem.

Kind regards
Frans Meerhoff


sudo add-apt-repository ppa:jasp-uva/jasp
sudo apt-get update
sudo apt-get install jasp

For version 0.8.0.0, extra R packages have to be installed:

sudo R
install.packages(c("GPArotation","qgraph","RJSONIO","semTools"))

**Comment by FransMeerhoff** [comment]
Hello @guidocor 

That's odd, but thanks for helping us anyway.
Could you try the following to find out whats going on:

- Find location of the packages: 
After installation of R its packages reside mostly in /usr/lib/R/library
New installed packages in /usr/local/lib/R/site-library
Could you check if the psych package is there by typing in a terminal
$ ls  /usr/local/lib/R/site-library/psych
This should give some info of the psych content

- start JASP from a terminal and redirect the result to some output file
$ JASP > ~/out.txt
Close JASP and send us the out.txt

- Information about your environment
From a terminal type
$ set > ~/set.txt
Send us the set.txt

These files can be added to this issue by dragging and dropping them.
Hope this will give us some extra information to solve your issue.

Kind regards
Frans Meerhoff

**Comment by lindeloev** [comment]
I have the same problem with JASP 0.8.2 on Ubuntu 16.04. My R packages are in `~/.R/` and `/usr/local/lib/R/site-library/` is empty.

Error when doing EFA on the Big5 example data:

```
This analysis terminated unexpectedly.

Error in loadNamespace(name): there is no package called ‘psych’

Stack trace
analysis(dataset = NULL, options = options, perform = perform, callback = the.callback, state = state)

psych::fa.parallel

getExportedValue(pkg, name)

asNamespace(ns)

getNamespace(ns)

tryCatch(loadNamespace(name), error = function(e) stop(e))

tryCatchList(expr, classes, parentenv, handlers)

tryCatchOne(expr, names, parentenv, handlers[[1]])

value[[3]](cond)

To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports
```

**Comment by FransMeerhoff** [comment]
Hi @guidocor @lindeloev 

Since version JASP 0.8.1 we support R  version R 3.3
If you have an older version of R running, could you update R and 
check if the problem is solved?

Kind regards
Frans Meerhoff

**Comment by lindeloev** [comment]
I have R 3.4.1 and JASP 0.8.2. Same error. 

The library paths include `~/.R/` where `psych` package is installed, so maybe JASP is hard-coded to a particular directory instead of using `libPaths`?
```
> .libPaths()
[1] "/home/jonas/.R"                "/usr/local/lib/R/site-library"
[3] "/usr/lib/R/site-library"       "/usr/lib/R/library"
```

**Comment by FransMeerhoff** [comment]
Hi @lindeloev 

Unfortunately I'm still not able to reproduce this issue on a clean Ubuntu 16.04.3 machine also with R 3.4.1
Could you please start JASP from a terminal by simple by typing
~$ JASP
JASP now produces some output where all packages and versions are checked.
We are of course only interested in the psych package.
Can you find out what JASP says on that line there?

BTW this is a minimal list of packages needed to run JASP (without any modules)

install.packages(c('BAS', 'BayesFactor', 'SuppDists', 'afex', 'car', 'conting', 'ggplot2', 'hypergeo', 'lavaan','logspline', 'lsmeans', 'multcomp', 'plotrix', 'plyr', 'psych', 'qgraph', 'rjson', 'semTools', 'stringr', 'vcd','vcdExtra'), repos = 'https://cloud.r-project.org', dependencies = NA)

Kind regards
Frans Meerhoff

**Comment by lindeloev** [comment]
Sure, the output is below. I then tried firing up R from the console (instead of RStudio which I usually used) and running your install command. There were a few errors/warnings:

```
Warning messages:
1: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘curl’ had non-zero exit status
2: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘TTR’ had non-zero exit status
3: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘igraph’ had non-zero exit status
4: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘quantmod’ had non-zero exit status
5: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘huge’ had non-zero exit status
6: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘ggm’ had non-zero exit status
7: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘BDgraph’ had non-zero exit status
8: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘tseries’ had non-zero exit status
9: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘qgraph’ had non-zero exit status
10: In install.packages(c("BAS", "BayesFactor", "SuppDists", "afex",  :
  installation of package ‘conting’ had non-zero exit status
```

### Before running install command
```
jonas@jonas-ThinkPad-P50:~$ JASP 
QObject::connect: No such signal FSBModel::hideAuthentication()
QObject::connect: No such signal FSBModel::hideAuthentication()
QObject::connect: No such signal FSBModel::hideAuthentication()
QObject::connect:  (receiver name: 'browser')
QObject::connect: No such signal FSBMExamples::hideAuthentication()
subprocess started
subprocess started
subprocess started
subprocess started
{
   "messages" : [
      "Package AER not installed!",
      "Package Amelia not installed!",
      "Package BB not installed!",
      "Package BH not installed!",
      "Package BMS not installed!",
      "Package CARBayes not installed!",
      "Package CARBayesdata not installed!",
      "Package DBI not installed!",
      "Package DEoptimR not installed!",
      "Package DiagrammeR not installed!",
      "Package Fahrmeir not installed!",
      "Package Formula not installed!",
      "Package GPArotation not installed!",
      "Package HSAUR not installed!",
      "Package HSAUR2 not installed!",
      "Package Hmisc not installed!",
      "Package ISwR not installed!",
      "Package LearnBayes not installed!",
      "Package MASS is not the correct version; expected: 7.3-45, installed: 7.3-47",
      "Package MBESS not installed!",
      "Package MCMCglmm not installed!",
      "Package MCMCpack not installed!",
      "Package MEMSS not installed!",
      "Package Matrix is not the correct version; expected: 1.2-8, installed: 1.2-10",
      "Package NMF not installed!",
      "Package OpenMx not installed!",
      "Package PKPDmodels not installed!",
      "Package R.cache not installed!",
      "Package R.methodsS3 not installed!",
      "Package R.oo not installed!",
      "Package R.rsp not installed!",
      "Package R.utils not installed!",
      "Package R6 is not the correct version; expected: 2.2.0, installed: 2.1.2",
      "Package RJSONIO not installed!",
      "Package RSVGTipsDevice not installed!",
      "Package RUnit not installed!",
      "Package Rcgmin not installed!",
      "Package Rcmdr not installed!",
      "Package RcmdrMisc not installed!",
      "Package Rcpp is not the correct version; expected: 0.12.10, installed: 0.12.3",
      "Package RcppArmadillo not installed!",
      "Package RcppEigen is not the correct version; expected: 0.3.2.9.1, installed: 0.3.2.7.0",
      "Package Rcsdp not installed!",
      "Package Rook not installed!",
      "Package Rvmmin not installed!",
      "Package SimComp not installed!",
      "Package Sleuth2 not installed!",
      "Package SparseM is not the correct version; expected: 1.76, installed: 1.7",
      "Package StanHeaders not installed!",
      "Package SuppDists not installed!",
      "Package TH.data is not the correct version; expected: 1.0-8, installed: 1.0-7",
      "Package VGAM not installed!",
      "Package XML not installed!",
      "Package abind is not the correct version; expected: 1.4-5, installed: 1.4-3",
      "Package acepack not installed!",
      "Package afex is not the correct version; expected: 0.16-1, installed: 0.15-2",
      "Package alr4 not installed!",
      "Package ape not installed!",
      "Package arm not installed!",
      "Package ascii not installed!",
      "Package assertthat not installed!",
      "Package backports not installed!",
      "Package base is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package base64enc is not the correct version; expected: 0.1-3, installed: 0.1-2",
      "Package bdsmatrix not installed!",
      "Package boot is not the correct version; expected: 1.3-18, installed: 1.3-20",
      "Package brew not installed!",
      "Package ca not installed!",
      "Package caTools not installed!",
      "Package calibrator not installed!",
      "Package car is not the correct version; expected: 2.1-4, installed: 2.1-1",
      "Package checkmate not installed!",
      "Package coda is not the correct version; expected: 0.19-1, installed: 0.18-1",
      "Package coin is not the correct version; expected: 1.1-3, installed: 1.1-2",
      "Package colorspace is not the correct version; expected: 1.3-2, installed: 1.2-4",
      "Package compiler is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package contfrac is not the correct version; expected: 1.1-10, installed: 1.1-9",
      "Package conting not installed!",
      "Package corpcor not installed!",
      "Package covr not installed!",
      "Package coxme not installed!",
      "Package crayon is not the correct version; expected: 1.3.2, installed: 1.3.1",
      "Package cubature not installed!",
      "Package curl not installed!",
      "Package d3Network not installed!",
      "Package data.table not installed!",
      "Package datasets is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package deSolve is not the correct version; expected: 1.14, installed: 1.12",
      "Package deldir not installed!",
      "Package dfoptim not installed!",
      "Package digest is not the correct version; expected: 0.6.12, installed: 0.6.9",
      "Package diptest not installed!",
      "Package doParallel not installed!",
      "Package dplyr not installed!",
      "Package e1071 not installed!",
      "Package effects is not the correct version; expected: 3.1-2, installed: 3.0-6",
      "Package ellipse not installed!",
      "Package elliptic is not the correct version; expected: 1.3-7, installed: 1.3-5",
      "Package emulator not installed!",
      "Package estimability is not the correct version; expected: 1.2, installed: 1.1-1",
      "Package evaluate not installed!",
      "Package expm not installed!",
      "Package fdrtool not installed!",
      "Package flexmix not installed!",
      "Package foreign is not the correct version; expected: 0.8-67, installed: 0.8-69",
      "Package fpc not installed!",
      "Package gamm4 not installed!",
      "Package gbm not installed!",
      "Package gdata not installed!",
      "Package gdtools not installed!",
      "Package gee not installed!",
      "Package geepack not installed!",
      "Package ggm not installed!",
      "Package ggplot2 is not the correct version; expected: 2.2.1, installed: 2.0.0",
      "Package ggplot2movies not installed!",
      "Package glasso not installed!",
      "Package gmodels not installed!",
      "Package grDevices is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package graphics is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package grid is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package gridBase not installed!",
      "Package gridExtra not installed!",
      "Package gsl not installed!",
      "Package gtable is not the correct version; expected: 0.2.0, installed: 0.1.2",
      "Package heplots not installed!",
      "Package hexbin not installed!",
      "Package highr not installed!",
      "Package hms not installed!",
      "Package htmlTable not installed!",
      "Package htmltools not installed!",
      "Package htmlwidgets not installed!",
      "Package httpuv not installed!",
      "Package httr not installed!",
      "Package huge not installed!",
      "Package hypergeo is not the correct version; expected: 1.2-13, installed: 1.2-11",
      "Package igraph not installed!",
      "Package influenceR not installed!",
      "Package inline not installed!",
      "Package irlba not installed!",
      "Package itertools not installed!",
      "Package jpeg not installed!",
      "Package jsonlite not installed!",
      "Package kernlab not installed!",
      "Package kknn not installed!",
      "Package knitr not installed!",
      "Package languageR not installed!",
      "Package lattice is not the correct version; expected: 0.20-34, installed: 0.20-35",
      "Package latticeExtra not installed!",
      "Package lavaan not installed!",
      "Package lazyeval not installed!",
      "Package leaps not installed!",
      "Package lme4 is not the correct version; expected: 1.1-12, installed: 1.1-10",
      "Package lmerTest not installed!",
      "Package lmtest is not the correct version; expected: 0.9-35, installed: 0.9-34",
      "Package logspline is not the correct version; expected: 2.1.9, installed: 2.1.8",
      "Package lpSolve not installed!",
      "Package lsmeans is not the correct version; expected: 2.25-5, installed: 2.21-1",
      "Package maps is not the correct version; expected: 3.1.1, installed: 3.0.2",
      "Package maptools not installed!",
      "Package markdown not installed!",
      "Package matrixcalc not installed!",
      "Package mclust not installed!",
      "Package mcmc not installed!",
      "Package mediation not installed!",
      "Package methods is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package mgcv is not the correct version; expected: 1.8-17, installed: 1.8-18",
      "Package mi not installed!",
      "Package mice not installed!",
      "Package mime not installed!",
      "Package miniUI not installed!",
      "Package mlmRev not installed!",
      "Package mnormt not installed!",
      "Package mratios not installed!",
      "Package multcomp is not the correct version; expected: 1.4-6, installed: 1.4-4",
      "Package multcompView not installed!",
      "Package munsell is not the correct version; expected: 0.4.3, installed: 0.4.2",
      "Package mvtnorm is not the correct version; expected: 1.0-6, installed: 1.0-5",
      "Package network not installed!",
      "Package numDeriv not installed!",
      "Package openssl not installed!",
      "Package optextras not installed!",
      "Package optimx not installed!",
      "Package ordinal not installed!",
      "Package parallel is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package pbapply is not the correct version; expected: 1.3-2, installed: 1.1-3",
      "Package pbivnorm not installed!",
      "Package pbkrtest is not the correct version; expected: 0.4-7, installed: 0.4-6",
      "Package permute not installed!",
      "Package pixmap not installed!",
      "Package pkgKitten is not the correct version; expected: 0.1.4, installed: 0.1.3",
      "Package pkgmaker not installed!",
      "Package plotrix is not the correct version; expected: 3.6-4, installed: 3.6",
      "Package plyr is not the correct version; expected: 1.8.4, installed: 1.8.3",
      "Package png not installed!",
      "Package poLCA not installed!",
      "Package polycor not installed!",
      "Package prabclus not installed!",
      "Package praise not installed!",
      "Package pscl not installed!",
      "Package psych not installed!",
      "Package qgraph not installed!",
      "Package quadprog not installed!",
      "Package quantreg is not the correct version; expected: 5.29, installed: 5.21",
      "Package qvcalc is not the correct version; expected: 0.9-0, installed: 0.8-9",
      "Package randomForest not installed!",
      "Package readxl not installed!",
      "Package registry not installed!",
      "Package relimp is not the correct version; expected: 1.0-5, installed: 1.0-4",
      "Package reshape2 is not the correct version; expected: 1.4.2, installed: 1.4",
      "Package rex not installed!",
      "Package rgexf not installed!",
      "Package rgl not installed!",
      "Package rmarkdown not installed!",
      "Package rngtools not installed!",
      "Package robustbase not installed!",
      "Package rpart is not the correct version; expected: 4.1-10, installed: 4.1-11",
      "Package rpf not installed!",
      "Package rprojroot not installed!",
      "Package rsm not installed!",
      "Package rstudioapi not installed!",
      "Package rtiff not installed!",
      "Package scales is not the correct version; expected: 0.4.1, installed: 0.3.0",
      "Package scatterplot3d not installed!",
      "Package sem not installed!",
      "Package semTools not installed!",
      "Package sendplot not installed!",
      "Package setRNG not installed!",
      "Package shapefiles not installed!",
      "Package shiny not installed!",
      "Package shinyjs not installed!",
      "Package sna not installed!",
      "Package sourcetools not installed!",
      "Package sp is not the correct version; expected: 1.2-4, installed: 1.0-15",
      "Package spam not installed!",
      "Package spdep not installed!",
      "Package splines is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package statnet.common not installed!",
      "Package stats is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package stats4 is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package stringi is not the correct version; expected: 1.1.3, installed: 1.0-1",
      "Package stringr is not the correct version; expected: 1.2.0, installed: 1.0.0",
      "Package survey not installed!",
      "Package survival is not the correct version; expected: 2.40-1, installed: 2.41-3",
      "Package svUnit not installed!",
      "Package svglite not installed!",
      "Package tcltk is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package tcltk2 not installed!",
      "Package tensorA not installed!",
      "Package testthat not installed!",
      "Package tibble not installed!",
      "Package tools is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package trimcluster not installed!",
      "Package truncnorm not installed!",
      "Package tseries not installed!",
      "Package ucminf not installed!",
      "Package utils is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package vcd is not the correct version; expected: 1.4-3, installed: 1.4-1",
      "Package vcdExtra is not the correct version; expected: 0.7-0, installed: 0.6-11",
      "Package vegan not installed!",
      "Package viridis not installed!",
      "Package viridisLite not installed!",
      "Package visNetwork not installed!",
      "Package whisker not installed!",
      "Package withr not installed!",
      "Package xtable not installed!",
      "Package yaml not installed!",
      "Package zoo is not the correct version; expected: 1.7-14, installed: 1.7-12"
   ],
   "official" : false
```

### After running install command

```
jonas@jonas-ThinkPad-P50:~$ JASP
QObject::connect: No such signal FSBModel::hideAuthentication()
QObject::connect: No such signal FSBModel::hideAuthentication()
QObject::connect: No such signal FSBModel::hideAuthentication()
QObject::connect:  (receiver name: 'browser')
QObject::connect: No such signal FSBMExamples::hideAuthentication()
subprocess started
subprocess started
subprocess started
subprocess started
{
   "messages" : [
      "Package AER not installed!",
      "Package Amelia not installed!",
      "Package BB not installed!",
      "Package BH not installed!",
      "Package BMS not installed!",
      "Package CARBayes not installed!",
      "Package CARBayesdata not installed!",
      "Package DBI not installed!",
      "Package DEoptimR not installed!",
      "Package DiagrammeR not installed!",
      "Package Fahrmeir not installed!",
      "Package Formula not installed!",
      "Package GPArotation not installed!",
      "Package HSAUR not installed!",
      "Package HSAUR2 not installed!",
      "Package Hmisc not installed!",
      "Package ISwR not installed!",
      "Package LearnBayes not installed!",
      "Package MASS is not the correct version; expected: 7.3-45, installed: 7.3-47",
      "Package MBESS not installed!",
      "Package MCMCglmm not installed!",
      "Package MCMCpack not installed!",
      "Package MEMSS not installed!",
      "Package Matrix is not the correct version; expected: 1.2-8, installed: 1.2-10",
      "Package NMF not installed!",
      "Package OpenMx not installed!",
      "Package PKPDmodels not installed!",
      "Package R.cache not installed!",
      "Package R.methodsS3 not installed!",
      "Package R.oo not installed!",
      "Package R.rsp not installed!",
      "Package R.utils not installed!",
      "Package R6 is not the correct version; expected: 2.2.0, installed: 2.1.2",
      "Package RJSONIO not installed!",
      "Package RSVGTipsDevice not installed!",
      "Package RUnit not installed!",
      "Package Rcgmin not installed!",
      "Package Rcmdr not installed!",
      "Package RcmdrMisc not installed!",
      "Package Rcpp is not the correct version; expected: 0.12.10, installed: 0.12.3",
      "Package RcppArmadillo not installed!",
      "Package RcppEigen is not the correct version; expected: 0.3.2.9.1, installed: 0.3.2.7.0",
      "Package Rcsdp not installed!",
      "Package Rook not installed!",
      "Package Rvmmin not installed!",
      "Package SimComp not installed!",
      "Package Sleuth2 not installed!",
      "Package SparseM is not the correct version; expected: 1.76, installed: 1.7",
      "Package StanHeaders not installed!",
      "Package SuppDists not installed!",
      "Package TH.data is not the correct version; expected: 1.0-8, installed: 1.0-7",
      "Package VGAM not installed!",
      "Package XML not installed!",
      "Package abind is not the correct version; expected: 1.4-5, installed: 1.4-3",
      "Package acepack not installed!",
      "Package afex is not the correct version; expected: 0.16-1, installed: 0.15-2",
      "Package alr4 not installed!",
      "Package ape not installed!",
      "Package arm not installed!",
      "Package ascii not installed!",
      "Package assertthat not installed!",
      "Package backports not installed!",
      "Package base is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package base64enc is not the correct version; expected: 0.1-3, installed: 0.1-2",
      "Package bdsmatrix not installed!",
      "Package boot is not the correct version; expected: 1.3-18, installed: 1.3-20",
      "Package brew not installed!",
      "Package ca not installed!",
      "Package caTools not installed!",
      "Package calibrator not installed!",
      "Package car is not the correct version; expected: 2.1-4, installed: 2.1-1",
      "Package checkmate not installed!",
      "Package coda is not the correct version; expected: 0.19-1, installed: 0.18-1",
      "Package coin is not the correct version; expected: 1.1-3, installed: 1.1-2",
      "Package colorspace is not the correct version; expected: 1.3-2, installed: 1.2-4",
      "Package compiler is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package contfrac is not the correct version; expected: 1.1-10, installed: 1.1-9",
      "Package conting not installed!",
      "Package corpcor not installed!",
      "Package covr not installed!",
      "Package coxme not installed!",
      "Package crayon is not the correct version; expected: 1.3.2, installed: 1.3.1",
      "Package cubature not installed!",
      "Package curl not installed!",
      "Package d3Network not installed!",
      "Package data.table not installed!",
      "Package datasets is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package deSolve is not the correct version; expected: 1.14, installed: 1.12",
      "Package deldir not installed!",
      "Package dfoptim not installed!",
      "Package digest is not the correct version; expected: 0.6.12, installed: 0.6.9",
      "Package diptest not installed!",
      "Package doParallel not installed!",
      "Package dplyr not installed!",
      "Package e1071 not installed!",
      "Package effects is not the correct version; expected: 3.1-2, installed: 3.0-6",
      "Package ellipse not installed!",
      "Package elliptic is not the correct version; expected: 1.3-7, installed: 1.3-5",
      "Package emulator not installed!",
      "Package estimability is not the correct version; expected: 1.2, installed: 1.1-1",
      "Package evaluate not installed!",
      "Package expm not installed!",
      "Package fdrtool not installed!",
      "Package flexmix not installed!",
      "Package foreign is not the correct version; expected: 0.8-67, installed: 0.8-69",
      "Package fpc not installed!",
      "Package gamm4 not installed!",
      "Package gbm not installed!",
      "Package gdata not installed!",
      "Package gdtools not installed!",
      "Package gee not installed!",
      "Package geepack not installed!",
      "Package ggm not installed!",
      "Package ggplot2 is not the correct version; expected: 2.2.1, installed: 2.0.0",
      "Package ggplot2movies not installed!",
      "Package glasso not installed!",
      "Package gmodels not installed!",
      "Package grDevices is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package graphics is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package grid is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package gridBase not installed!",
      "Package gridExtra not installed!",
      "Package gsl not installed!",
      "Package gtable is not the correct version; expected: 0.2.0, installed: 0.1.2",
      "Package heplots not installed!",
      "Package hexbin not installed!",
      "Package highr not installed!",
      "Package hms not installed!",
      "Package htmlTable not installed!",
      "Package htmltools not installed!",
      "Package htmlwidgets not installed!",
      "Package httpuv not installed!",
      "Package httr not installed!",
      "Package huge not installed!",
      "Package hypergeo is not the correct version; expected: 1.2-13, installed: 1.2-11",
      "Package igraph not installed!",
      "Package influenceR not installed!",
      "Package inline not installed!",
      "Package irlba not installed!",
      "Package itertools not installed!",
      "Package jpeg not installed!",
      "Package jsonlite not installed!",
      "Package kernlab not installed!",
      "Package kknn not installed!",
      "Package knitr not installed!",
      "Package languageR not installed!",
      "Package lattice is not the correct version; expected: 0.20-34, installed: 0.20-35",
      "Package latticeExtra not installed!",
      "Package lavaan not installed!",
      "Package lazyeval not installed!",
      "Package leaps not installed!",
      "Package lme4 is not the correct version; expected: 1.1-12, installed: 1.1-10",
      "Package lmerTest not installed!",
      "Package lmtest is not the correct version; expected: 0.9-35, installed: 0.9-34",
      "Package logspline is not the correct version; expected: 2.1.9, installed: 2.1.8",
      "Package lpSolve not installed!",
      "Package lsmeans is not the correct version; expected: 2.25-5, installed: 2.21-1",
      "Package maps is not the correct version; expected: 3.1.1, installed: 3.0.2",
      "Package maptools not installed!",
      "Package markdown not installed!",
      "Package matrixcalc not installed!",
      "Package mclust not installed!",
      "Package mcmc not installed!",
      "Package mediation not installed!",
      "Package methods is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package mgcv is not the correct version; expected: 1.8-17, installed: 1.8-18",
      "Package mi not installed!",
      "Package mice not installed!",
      "Package mime not installed!",
      "Package miniUI not installed!",
      "Package mlmRev not installed!",
      "Package mnormt not installed!",
      "Package mratios not installed!",
      "Package multcomp is not the correct version; expected: 1.4-6, installed: 1.4-4",
      "Package multcompView not installed!",
      "Package munsell is not the correct version; expected: 0.4.3, installed: 0.4.2",
      "Package mvtnorm is not the correct version; expected: 1.0-6, installed: 1.0-5",
      "Package network not installed!",
      "Package numDeriv not installed!",
      "Package openssl not installed!",
      "Package optextras not installed!",
      "Package optimx not installed!",
      "Package ordinal not installed!",
      "Package parallel is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package pbapply is not the correct version; expected: 1.3-2, installed: 1.1-3",
      "Package pbivnorm not installed!",
      "Package pbkrtest is not the correct version; expected: 0.4-7, installed: 0.4-6",
      "Package permute not installed!",
      "Package pixmap not installed!",
      "Package pkgKitten is not the correct version; expected: 0.1.4, installed: 0.1.3",
      "Package pkgmaker not installed!",
      "Package plotrix is not the correct version; expected: 3.6-4, installed: 3.6",
      "Package plyr is not the correct version; expected: 1.8.4, installed: 1.8.3",
      "Package png not installed!",
      "Package poLCA not installed!",
      "Package polycor not installed!",
      "Package prabclus not installed!",
      "Package praise not installed!",
      "Package pscl not installed!",
      "Package psych not installed!",
      "Package qgraph not installed!",
      "Package quadprog not installed!",
      "Package quantreg is not the correct version; expected: 5.29, installed: 5.21",
      "Package qvcalc is not the correct version; expected: 0.9-0, installed: 0.8-9",
      "Package randomForest not installed!",
      "Package readxl not installed!",
      "Package registry not installed!",
      "Package relimp is not the correct version; expected: 1.0-5, installed: 1.0-4",
      "Package reshape2 is not the correct version; expected: 1.4.2, installed: 1.4",
      "Package rex not installed!",
      "Package rgexf not installed!",
      "Package rgl not installed!",
      "Package rmarkdown not installed!",
      "Package rngtools not installed!",
      "Package robustbase not installed!",
      "Package rpart is not the correct version; expected: 4.1-10, installed: 4.1-11",
      "Package rpf not installed!",
      "Package rprojroot not installed!",
      "Package rsm not installed!",
      "Package rstudioapi not installed!",
      "Package rtiff not installed!",
      "Package scales is not the correct version; expected: 0.4.1, installed: 0.3.0",
      "Package scatterplot3d not installed!",
      "Package sem not installed!",
      "Package semTools not installed!",
      "Package sendplot not installed!",
      "Package setRNG not installed!",
      "Package shapefiles not installed!",
      "Package shiny not installed!",
      "Package shinyjs not installed!",
      "Package sna not installed!",
      "Package sourcetools not installed!",
      "Package sp is not the correct version; expected: 1.2-4, installed: 1.0-15",
      "Package spam not installed!",
      "Package spdep not installed!",
      "Package splines is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package statnet.common not installed!",
      "Package stats is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package stats4 is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package stringi is not the correct version; expected: 1.1.3, installed: 1.0-1",
      "Package stringr is not the correct version; expected: 1.2.0, installed: 1.0.0",
      "Package survey not installed!",
      "Package survival is not the correct version; expected: 2.40-1, installed: 2.41-3",
      "Package svUnit not installed!",
      "Package svglite not installed!",
      "Package tcltk is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package tcltk2 not installed!",
      "Package tensorA not installed!",
      "Package testthat not installed!",
      "Package tibble not installed!",
      "Package tools is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package trimcluster not installed!",
      "Package truncnorm not installed!",
      "Package tseries not installed!",
      "Package ucminf not installed!",
      "Package utils is not the correct version; expected: 3.3.3, installed: 3.4.1",
      "Package vcd is not the correct version; expected: 1.4-3, installed: 1.4-1",
      "Package vcdExtra is not the correct version; expected: 0.7-0, installed: 0.6-11",
      "Package vegan not installed!",
      "Package viridis not installed!",
      "Package viridisLite not installed!",
      "Package visNetwork not installed!",
      "Package whisker not installed!",
      "Package withr not installed!",
      "Package xtable not installed!",
      "Package yaml not installed!",
      "Package zoo is not the correct version; expected: 1.7-14, installed: 1.7-12"
   ],
   "official" : false
}

received message
child not found :  "highlightSlider"  in AnalysisForm::setOptions()
child not found :  "incl_loadings"  in AnalysisForm::setOptions()
child not found :  "incl_GoF"  in AnalysisForm::setOptions()
child not found :  "plotWidthPathDiagram"  in AnalysisForm::setOptions()
child not found :  "plotHeightPathDiagram"  in AnalysisForm::setOptions()
child not found :  "plotWidthScreePlot"  in AnalysisForm::setOptions()
child not found :  "plotHeightScreePlot"  in AnalysisForm::setOptions()
received message
received message
received message
```

**Comment by FransMeerhoff** [comment]
Hi @guidocor 

We have just update the ppa for JASP version 0.8.3.1.
Could you check if you still have the same problem?

Kind regards
Frans Meerhoff

**Comment by lindeloev** [comment]
Maybe you could solve this issue by providing `r-cran-psych` through the ppa.

**Comment by lindeloev** [comment]
I looked a bit more into this and I thin that I identified the problem! JASP doesn't source all `.libPaths`. On my system, it only looks at `/usr/lib/R/site-library`. The packages there seem to be purely installed from some repository with outdated packages, e.g. `lme4` version 1.1-10 from 2015. When I pasted `psych` into this, EFA stopped giving the `'psych'` error, now progressing to `mnormt`.

I don't know why it looks at this specific path. It's the third in `.libPaths`:

```
> .libPaths()
[1] "/home/jonas/.R"                "/usr/local/lib/R/site-library"
[3] "/usr/lib/R/site-library"       "/usr/lib/R/library"
```

To play with it, I removed everything in `/usr/lib/R/site-library` which caused JASP to crash at startup with the error message `""`. Then I purged JASP and the R dependencies and ran `sudo apt-get install jasp` again, which filled `usr/lib/R/site-library` again. But JASP fails with the same GUI error. Launching from console, it shows the error `unable to open base package`.

So JASP is broken on my system now, but maybe this will give you a lead to fix it.

**Comment by lindeloev** [comment]
This error persists in 0.8.4. Did you see my debugging of the problem in the comment above?

**Comment by FransMeerhoff** [comment]
Hi @lindeloev 

JASP is now available for virtually all Linux distributions through the widely available flatpak. This is most likely already installed on your distribution and otherwise can be  easily acquired through your package manager.

We refer to our download page for further installation info and close this issue.

Kind regards
Frans Meerhoff

**Comment by lindeloev** [comment]
@FransMeerhoff, Flatpak doesn't work on my Ubuntu 16.04 but that was addressed in #2423. TL;DR: There's no solution but to upgrade to a newer Ubuntu. 

That's OK; I'm just mentioning it here if others look to this issue for solutions.

**Comment by AlexanderLyNL** [comment]
Thanks for the additional info.
